numpyro/distributions/constraints.py

```diff
@@ -369,6 +371,19 @@ def feasible_like(self, prototype):
         return jax.numpy.full_like(prototype, 1 / prototype.shape[-1])
 
 
+class _SoftplusPositive(_GreaterThan):
+    def __init__(self):
+        super().__init__(lower_bound=0.0)
+
+    def feasible_like(self, prototype):
+        return jax.numpy.full(jax.numpy.shape(prototype), np.log(2))
```

Comment:
nit: should we scale softplus transform so that 0 -> 1 (instead of log(2))?

Reply:
pymc3 initialises to 0, such that `scales = softplus(0)`

https://discourse.pymc.io/t/how-to-choose-initial-values-for-meanfield-sigma-and-transform-rho-sigma/6343/2?u=vitkl

Reply:
Interesting! Currently, the behavior in Pyro/NumPyro is to init unconstrained scale parameter to `softplus_inv(0.1)` or `exp_inv(0.1)`. This feasible_like method is only useful to initialize latent variables (e.g. in HMC). I think having 0 -> 1 here is not important...